app/uk/gov/hmrc/gatekeeper/views/MainView.scala.html

```diff
@@ -42,7 +42,7 @@
 )(
   mainContent: Html
 )(
-  implicit loggedInUser: LoggedInUser = LoggedInUser(None),
+  implicit loggedInUser: LoggedInUser = LoggedInUser("Bobby Example"),
```

Comment:
is this wanted in the real mainView?

Reply:
That's a really good question.  I think you've made me remember why LIU was a handy option wrapper - over and above the fact that stride auth always returns an option string (which is always populated if user logs in even though the auth type can't guarantee this (hence the .get everywhere).  But this is the one place that should be Option(LIU) where LUI isn't optional internally.  This will need some rework.  Drat.  Thanks for spotting this.

Reply:
Oh - and we also were putting bobby example in tests.... so it's even more of a mistake.